pages/index.tsx

```diff
@@ -38,6 +38,10 @@ const Home: FC<HomeProps> = ({ faqList }) => {
     method: 'name',
   });
 
+  useEffect(() => {
+    fetch('api/oneinch-rate');
```

Comment:
Why do we need to fetch it without using response?

Reply:
this 'fetch' is needed for example to show how cache-control works in the oneinch API method.

Reply:
done